src/Protocol/Protocol_1_13.cpp

```diff
@@ -368,6 +368,34 @@ void cProtocol_1_13::SendScoreboardObjective(const AString & a_Name, const AStri
 
 
 
+void cProtocol_1_13::SendSpawnMob(const cMonster & a_Mob)
+{
+	ASSERT(m_State == 3);  // In game mode?
+
+	cPacketizer Pkt(*this, pktSpawnMob);
+	Pkt.WriteVarInt32(a_Mob.GetUniqueID());
+	// TODO: Bad way to write a UUID, and it's not a true UUID, but this is functional for now.
+	Pkt.WriteBEUInt64(0);
+	Pkt.WriteBEUInt64(a_Mob.GetUniqueID());
+	Pkt.WriteVarInt32(static_cast<UInt32>(a_Mob.GetMobType()));
```

Comment:
IMO we should have a `GetProtocolMobID` call here as well and the values of `eMonsterType` shouldn't be linked to any protocol version.

Reply:
I agree. What would be a good numbering scheme for eMonsterType?

Reply:
If we want the least changes to code, and maintain compatibility with old hardcoded plugins, then just keep the values the same. If we want to break compatibility, then just ordering the mobs in the order they were introduced to the game seems to make sense.

Reply:
Keeping them the same as before is probably the best idea. There's always going to be issues when we eventually get to implementing mobs that were introduced before already implemented ones.

Reply:
I would say either keep them the same, or don't specify values at all.